src/entitysdk/config.py

```diff
@@ -0,0 +1,15 @@
+"""Configuration for this library."""
+
+from typing import Annotated
+
+from pydantic import Field
+from pydantic_settings import BaseSettings
+
+
+class Constants(BaseSettings):
+    """Constants for this library."""
+
+    page_size: Annotated[int, Field(description="Default pagination page size.")] = 20
```

Comment:
This can be overridden using an env variable, but the env var name should be more specific, something like `ENTITYSDK_PAGE_SIZE` for example.
It should be enough to use `_env_prefix` from https://docs.pydantic.dev/latest/api/pydantic_settings/#pydantic_settings.BaseSettings

Reply:
I added the name explicitly. It is a little more readable than having a prefix.